This patch changes the configuration so the daemon runs as root instead and also
allows any client to connect.

If you don't like it, choose other settings.
Index: a/owntone.conf.in
===================================================================
--- a/owntone.conf.in
+++ b/owntone.conf.in
@@ -11,7 +11,7 @@ general {
 	# Username
 	# Make sure the user has read access to the library directories you set
 	# below, and full access to the databases, log and local audio
-	uid = "@OWNTONE_USER@"
+	uid = "root"
 
 	# Database location
 #	db_path = "@localstatedir@/cache/@PACKAGE@/songs3.db"
